conan/conanfile_windows.txt

```diff
@@ -0,0 +1,10 @@
+[requires]
+gtest/1.15.0
+tinyxml2/10.0.0
+npcap/1.70
```

Comment:
Version mismatch with readme ?

Reply:
Because the driver installed for the test was the 1.80 (with the npcap 1.70 SDK). Overall it is working with this configuration :)

Reply:
README updated to reflect the difference between the SDK (conan) and the driver (to be installed by the user)